Remove 'Add tooltip' docs page

janfaracik opened the pull request:
This page is now inaccurate and has been for a while, better to rely on the Design Library plugin for up to date information on working with Jenkins UI.

zbynek commented:
Also needs to be removed from the chapter https://github.com/jenkins-infra/jenkins.io/blob/c300643b0addbfae0451c8439e32a91b5fac205f/content/doc/developer/forms/_chapter.yml

janfaracik commented:
> Also needs to be removed from the chapter https://github.com/jenkins-infra/jenkins.io/blob/c300643b0addbfae0451c8439e32a91b5fac205f/content/doc/developer/forms/_chapter.yml

Thanks - had assumed the list was dynamic.

MarkEWaite reviewed (approved):
Thanks very much @janfaracik and @zbynek 

Just in cases someone had an embedded link to the page, I've added a redirect to the tooltips section of the design library plugin on weekly.ci.jenkins.io.

I tested a local copy of the site and confirmed that the redirect worked as expected.

If you'd rather not have the redirect, you are welcome to discard the commit that I added to the pull request.

I plan to merge this after about 24 hours on the assumption that gives you enough time to object to the change that I submitted.

janfaracik commented:
> Thanks very much @janfaracik and @zbynek 
> 
> 
> 
> Just in case someone had an embedded link to the page, I've added a redirect to the tooltips section of the design library plugin on weekly.ci.jenkins.io.
> 
> 
> 
> I tested a local copy of the site and confirmed that the redirect worked as expected.
> 
> 
> 
> If you'd rather not have the redirect, you are welcome to discard the commit that I added to the pull request.
> 
> 
> 
> I plan to merge this after about 24 hours on the assumption that gives you enough time to object to the change that I submitted.

That's great, thanks Mark. Appreciate it
